package.json

```diff
@@ -114,7 +114,7 @@
     "jest": "^27.4.0",
     "jest-axe": "^5.0.1",
     "jest-cli": "^27.4.0",
-    "jest-styled-components": "^7.0.5",
```

Comment:
took out the carrot because when I ran `yarn` it was trying to update these I have them in a separate PR